crates/flox-rust-sdk/src/models/flox_package.rs

```diff
@@ -211,9 +211,28 @@ pub enum ParseError {
     NoPackage,
 }
 
+impl Display for FloxTriple {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.stability != Stability::Stable {
+            write!(f, "{}.", self.stability)?;
+        }
+        if self.channel != "nixpkgs-flox" {
+            write!(f, "{}.", self.channel)?;
+        }
```

Comment:
I wonder if we should print packages always in normalized form, rather than stripping that information because during parsing its inferred.

By no means a need to block here given the feature flag, but @ghudgins or @jennymahmoudi might have an opinion.

Reply:
printing where, in the terminal? if I `flox install blah` then you're saying we print `installed stable.blah`?

Reply:
yes, flox (un)install as well as probably list and such

should they aim for the _shortest_ representation possible or the _least ambiguous_

Reply:
@ysndr - sorry for getting back to you so late. @jennymahmoudi and I met about this and feel we **should not** print the extra resolution information when installing. the declarative spec is also ambiguous and our main goal for the declarative spec is it is capturing user intentions and not the resolution behavior (we have a lock for the resolution). Because we don't want to see it in the manifest if it's not provided, we also don't want to show it in the CLI when installs are run.

Let us know if you have any questions about this!